sigma-schema.json

```
@@ -0,0 +1,246 @@
+              },
+              {
+                "const": "obsoletes",
+                "description": "The rule obsoletes the referred rule or rules, which aren't used anymore"
+              },
+              {
+                "const": "merged",
+                "description": "The rule was merged from the referred rules. The rules may be still existing and in use"
+              },
+              {
+                "const": "renamed",
+                "description": "The rule had previously the referred identifier or identifiers but was renamed for whatever reason, e.g. from a private naming scheme to UUIDs, to resolve collisions etc. It's not expected that a rule with this id exists anymore"
+              },
+              {
+                "const": "similar",
+                "description": "TUse to relate similar rules to each other (e.g. same detection content applied to different log sources, rule that is a modified version of another rule with a different level)"
+              }
+            ]
+          }
+        }
+      }
+    },
+    "status": {
+      "type": "string",
+      "oneOf": [
+        {
+          "const": "stable",
+          "description": "The rule didn't produce any obvious false positives in multiple environments over a long period of time"
+        },
+        {
+          "const": "test",
+          "description": "The rule doesn't show any obvious false positives on a limited set of test systems"
+        },
+        {
+          "const": "experimental",
+          "description": "A new rule that hasn't been tested outside of lab environments and could lead to many false positives"
+        },
+        {
+          "const": "deprecated",
+          "description": "The rule is to replace or cover another one. The link between both rules is made via the `related` field"
+        },
+        {
+          "const": "unsupported",
+          "description": "The rule can not be used in its current state (special correlation log, home-made fields...etc.)"
+        }
+      ]
+    },
+    "description": {
+      "type": "string",
+      "description": "A short description of the rule and the malicious activity that can be detected",
+      "maxLength": 65535
+    },
+    "license": {
+      "type": "string",
+      "description": "License of the rule according the SPDX ID specification (https://spdx.dev/ids/)"
+    },
+    "author": {
+      "type": "string",
+      "description": "Creator of the rule. (can be a name, nickname, twitter handle, etc.)"
+    },
+    "references": {
+      "type": "array",
+      "description": "References to the source that the rule was derived from. These could be blog articles, technical papers, presentations or even tweets",
+      "items": {
+        "type": "string"
+      }
+    },
+    "date": {
+      "type": "string",
+      "description": "Creation date of the rule. Use the format YYYY/MM/DD",
+      "pattern": "^\\d{4}[/]\\d{2}[/]\\d{2}$"
```

I would prefer a more strict regex as dates are defined. Here is a suggestion from StackOF. If you have a better or stricter one please use it.
```suggestion
      "pattern": "^\\d{4}\\-(0?[1-9]|1[012])\\-(0?[1-9]|[12][0-9]|3[01])$"
```

---

edit by me: I'd enforce the leading zeroes. And we use `/` instead of `-`.

---

(maybe something we change with spec v2)

---

I already fixed it